concent_api/api-integration-additional-verification-test.py

```diff
@@ -0,0 +1,367 @@
+    print('\nUploaded file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(
+        task_id,
+        file_check_sum_1,
+        file_size_1
+    ))
+
+    response = upload_file_to_storage_cluster(
+        file_content_2,
+        ack_subtask_results_verify.file_transfer_token.files[1]['path'],
+        ack_subtask_results_verify.file_transfer_token
+    )
+    assert_condition(response.status_code, 200, 'File has not been stored on cluster')
+    print('\nUploaded file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(
+        task_id,
+        file_check_sum_2,
+        file_size_2
+    ))
+
+    time.sleep(10)
+
+    api_request(
+        cluster_url,
+        'receive-out-of-band',
+        PROVIDER_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        create_client_auth_message(PROVIDER_PRIVATE_KEY, PROVIDER_PUBLIC_KEY, CONCENT_PUBLIC_KEY),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.ForceSubtaskResultsResponse.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+    api_request(
+        cluster_url,
+        'receive-out-of-band',
+        REQUESTOR_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        create_client_auth_message(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY, CONCENT_PUBLIC_KEY),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.ForceSubtaskResultsResponse.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+
+@count_fails
+def test_case_4_timeout(cluster_consts, cluster_url, test_id):
+    current_time    = get_current_utc_timestamp()
+    (subtask_id, task_id) = get_task_id_and_subtask_id(test_id, 'existing_file')
+
+    file_content = task_id
+    file_size = len(file_content)
+    file_check_sum = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
+
+    api_request(
+        cluster_url,
+        'send',
+        REQUESTOR_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        get_subtask_results_verify(
+            task_id,
+            subtask_id,
+            current_time - cluster_consts.additional_verification_call_time,
+            cluster_consts,
+            reason=message.tasks.SubtaskResultsRejected.REASON.VerificationNegative,
+            report_computed_task_size=file_size,
+            report_computed_task_package_hash=file_check_sum,
+            task_to_compute_size=file_size,
+            task_to_compute_package_hash=file_check_sum,
+        ),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.AckSubtaskResultsVerify.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+    time.sleep(1)
```

Comment:
`current_time - cluster_consts.additional_verification_call_time` this suffice to get the timeout, but only if `cluster_consts.additional_verification_call_time < GOLEM_MESSAGE_TIME_BEFORE_MESSAGE_TOO_OLD_IS_RAISED`. Otherwise we would have to put `cluster_consts.additional_verification_call_time` in the `sleep()`. So the settings has to be `specific` for this case. On the other hand, we can just simply not test it.

Reply:
After discuss with @bartoszbetka it looks like there should be no problem with that.